src/main/scala/org/broadinstitute/dsde/firecloud/filematch/strategy/OntSingleReadStrategy.scala

```diff
@@ -0,0 +1,21 @@
+package org.broadinstitute.dsde.firecloud.filematch.strategy
+
+import org.broadinstitute.dsde.firecloud.filematch.result.{FailedMatchResult, FileMatchResult, SuccessfulMatchResult}
+import OntSingleReadStrategy.PATTERN
+
+import java.nio.file.{Path, Paths}
+import scala.util.matching.Regex
+
+class OntSingleReadStrategy extends FileRecognitionStrategy {
+
+  override def matchFirstFile(path: Path): FileMatchResult =
+    PATTERN.findFirstMatchIn(path.toString) match {
+      case Some(regexMatch) => SuccessfulMatchResult(path, Paths.get(""), regexMatch.group(1))
+      case None             => FailedMatchResult(path)
+    }
+}
+
+object OntSingleReadStrategy {
+  // if the file contains barcode### and ends with .fastq or .fastq.gz, it's an ONT single read file
+  val PATTERN: Regex = "(?i)(.*barcode\\d+).*\\.fastq(?:.gz)*$".r
```

Comment:
I believe the `*` at the end should be a `?` or `Complete_barcode01.fastq.gz.gz.gz.gz.gz.gz` will match. 

You can also use `"""regex"""` to avoid having to escape your back slashes

Reply:
Done: https://github.com/broadinstitute/firecloud-orchestration/pull/1581/commits/84e78009d6a1a332e8fb08e51c98414837c3a902